spec/hbw/features/bp_form/translation_mock.yml

```diff
@@ -0,0 +1,211 @@
+## Structure of file:
```

Comment:
I intend to use one mock file for one _spec file
It will have next structure:
.../
  test_spec.rb
  test_mock.yml